ui/pages/bridge/quotes/multichain-bridge-quote-card.test.tsx

```diff
@@ -18,7 +18,7 @@ describe('MultichainBridgeQuoteCard', () => {
     jest.clearAllMocks();
```

Comment:
Do you think it might be good to also have an e2e that covers this flow @micaelae ? If so, we should create an issue for that, or add it here in this same PR. Whatever makes more sense, for you!

Reply:
thank you v much @micaelae Let me know if you need support from the QA team for that :pray: